backend/kesaseteli/requirements.in

```diff
@@ -12,4 +12,4 @@ mozilla-django-oidc
 psycopg2
 sentry-sdk
 requests
--e ../shared
+-e file:../shared
```

Comment:
is this intended?

Reply:
Yes. The reasoning is in the commit message: 
> Add the "file:" prefix so that pip-compile wont generate the absolute path instead of the relative path of the shared package.